tests/test-modules-import.php

```diff
@@ -160,7 +160,11 @@ public function test_wxrInsertAndFindTerm() {
 
 		// Clean attachments after test
 		$date = date( 'Y/m' );
-		array_map( 'unlink', array_filter( (array) glob( "/tmp/wordpress/wp-content/uploads/{$date}/*" ) ) );
```

Comment:
Sometimes pressbooks generate not only files but directories, so we need to check if the deleted files are files because unlink does not remove directories and PHP 7.4 throws a warning